cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ApplicationMetricDTO.java

```diff
@@ -60,14 +74,34 @@ public void setEndpoint( String endpoint )
         this.endpoint = endpoint;
     }
 
-    public FrequenceType getFrequence()
+    /**
+     * @deprecated use {@link #getFrequency()}
+     */
+    @Deprecated
+    public Frequency getFrequence()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)